MSAL/src/public/configuration/publicClientApplication/cache/MSALCacheConfig.h

```diff
@@ -84,9 +84,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
     Array of SecTrustedApplicationsRef that is allowed to access the keychain elements
-    created by the keychain cache.
+    created by the keychain cache on 10.12-10.14 devices.
+    createTrustedApplicationListFromPaths:error: helper API can be used to easily create a list of trustedApplications from a list of file paths.
  */
-@property (readonly, nonnull) NSArray *trustedApplications;
```

Comment:
Seems like bug that developer has no way to set the `trustedApplications` previously :P

Reply:
jepp, easily fixable though :)